### ISSUE TITLE
[Bug report] yarn build fails with local plugins on Windows, even use `const { path } = require('@vuepress/utils')`

### ISSUE THREAD

**Issue by LucienZhang** [vuepress/vuepress-next#443]
### Description

yarn build fails with local plugins on Windows, even use `const { path } = require('@vuepress/utils')`

### Steps to reproduce

init vuepress and enable a local plugin in `docs/.vuepress/config.js`, build it in windows-latest

yarn build fails because of `stack: 'ModuleBuildError: Module build failed: UnhandledSchemeError: Reading from "D:asnippetssnippetspackages@snippetors\x0Buepress-plugin-tabslibclient.js" is not handled by plugins (Unhandled scheme)`

Please note that the absolute path of the file is `D:\a\snippets\snippets\packages\@snippetors\vuepress-plugin-tabs\lib\client.js`, the backslash `\` incrrectly escapes the letter follows it (e.g. `\v` => `\x0B`)

- Reproduction link / repo:

https://github.com/Snippetors/snippets/pull/26

The CI job fails on Windows only. one thing suspicious is that `path.sep` is `/` in that windows runner, and the resolved path doesn't contain any backslashes `\`

### Expected behavior

It loads local plugins well as it works on Linux/MacOS

### Screenshots

Please check the failed CI in https://github.com/Snippetors/snippets/pull/26

### Environment info
it's the Action runner provided by GitHub, the config is [here](https://github.com/Snippetors/snippets/blob/main/.github/workflows/vuepress_check.yml)
Basically copied from this repo's [CI check](https://github.com/vuepress/vuepress-next/blob/main/.github/workflows/check.yml)

**Comment by meteorlxy** [comment]
The error was caused by your plugin. You didn't use `const { path } = require('@vuepress/utils')` in your plugin

**Comment by LucienZhang** [comment]
Thanks for quick answer!

For those who finally find this page:
Just search GLOABALLY, and replace every single `const path = require("path")` by `const { path } = require("@vuepress/utils")`
